### PR TITLE
Client: Passive scan base64 decoding and JWT support

### PR DESCRIPTION
**Pull request by psiinon**
## Overview
Add support for base64 decoding and JWT detection in browser storage.

Also corrected the names of a few files.

## Related Issues
https://github.com/zaproxy/zaproxy/pull/8218

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [x] Write tests
- [x] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

File: addOns/client/src/main/java/org/zaproxy/addon/client/pscan/ClientPassiveScanHelper.java [hunk 1]
```diff
@@ -32,7 +35,7 @@ public class ClientPassiveScanHelper {
 
     private static final Logger LOGGER = LogManager.getLogger(ClientPassiveScanHelper.class);
     private static final int MAX_HREFS_TO_CHECK = 1000;
-
+    private static final String ALLOWED_CONTROL_CHARACTERS = "\n\r\t";
```
[the remark this answers is not in the document]
Comment (psiinon) [reply addOns/client/src/main/java/org/zaproxy/addon/client/pscan/ClientPassiveScanHelper.java:38]: Yeah, that sounds better

File: addOns/client/client.gradle.kts [hunk 2]
```diff
@@ -42,6 +43,7 @@ crowdin {
 }
 
 dependencies {
+    zapAddOn("commonlib")
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/client/client.gradle.kts:46]: That's fine, I just didn't clue in.

File: addOns/client/src/main/java/org/zaproxy/addon/client/pscan/ClientPassiveScanHelper.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (psiinon) [reply addOns/client/src/main/java/org/zaproxy/addon/client/pscan/ClientPassiveScanHelper.java]: Changed.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**